### PR TITLE
lis: update to 2.0.26 and disable building test programs

### PR DESCRIPTION
**Pull request by 3rav**

### CODE REVIEW

File: mingw-w64-lis/PKGBUILD [hunk 1]
```diff
@@ -37,6 +37,7 @@ build() {
     --enable-quad \
     --enable-f90 \
     --enable-fortran \
+    --disable-test \
```
Comment (Biswa96) [inline mingw-w64-lis/PKGBUILD:40]: Tests are build fine. This may not be required.

Comment (3rav) [reply mingw-w64-lis/PKGBUILD:40]: This reduces compile time and resulting size

Comment (Biswa96) [reply mingw-w64-lis/PKGBUILD:40]: Some users may want to run the tests. Also there is a `check()` function. Compile time is not so important in small packages. Are you sure compiled tests binaries install in the resultant package?

Comment (3rav) [reply mingw-w64-lis/PKGBUILD:40]: 2.0.24: 340 files vs 2.0.26: 283 files

not "bin" and "share/examples/lis" folder in 2.0.26

Comment (lazka) [reply mingw-w64-lis/PKGBUILD:40]: fair enough..

**Pull request merged**